Code/Tools/AssetProcessor/native/AssetManager/assetScannerWorker.cpp

```diff
@@ -118,14 +124,28 @@ void AssetScannerWorker::ScanForSourceFiles(const ScanFolderInfo& scanFolderInfo
         }
         else if (!AssetUtilities::IsInCacheFolder(absPath.toUtf8().constData())) // Ignore files in the cache
         {
-            //Entry is a file
-            m_fileList.insert(AZStd::move(assetFileInfo));
+            // Entry is a file - add it to the check list which will be processed multi-threaded later.
+            m_checkForExclusionList.insert(AZStd::move(assetFileInfo));
         }
     }
 }
 
 void AssetScannerWorker::EmitFiles()
 {
+    // multi-threaded processing of regular expresisons.  Note that each call to IsFileExcluded
```

Comment:
expresisons -> expressions

Reply:
Fixed.  More like, removed.  We don't try to multihtread, its actually slower to do so.  The cuhnks of work are actually too small :(